src/h5web/toolbar/controls/DomainSlider/Thumb.tsx

```diff
@@ -0,0 +1,29 @@
+import { forwardRef, HTMLProps } from 'react';
+import { FiSkipBack, FiSkipForward } from 'react-icons/fi';
+import styles from './DomainSlider.module.css';
+
+type Props = HTMLProps<HTMLDivElement> & {
+  index: number;
+  isAutoMin: boolean;
+  isAutoMax: boolean;
+};
+
+const Thumb = forwardRef<HTMLDivElement, Props>((props, ref) => {
+  const { index, isAutoMin, isAutoMax, disabled, ...thumbProps } = props;
+  const isAuto = (isAutoMin && index === 0) || (isAutoMax && index === 1);
+
+  return (
+    <div
+      ref={ref}
+      {...thumbProps}
+      tabIndex={disabled ? -1 : 0}
+      data-auto={isAuto}
+    >
+      <div className={styles.thumbBtnLike}>
+        {isAuto && (index === 0 ? <FiSkipBack /> : <FiSkipForward />)}
```

Comment:
..and this one is simply `{isAuto && <Icon />}`.

Heck, you could even move out the `isAuto &&` out of the component if we want to keep the icon on the thumb